portmidi/0001-fix-build-with-Werror-format-security.patch

```diff
@@ -0,0 +1,117 @@
+From fa24b4246368c6a007b909a2754dbbbeba1c79e7 Mon Sep 17 00:00:00 2001
+From: cubercsl <2014cais01@gmail.com>
+Date: Tue, 11 Jan 2022 02:08:05 +0800
+Subject: [PATCH] fix: build with -Werror-format-security
+
+---
+ test/latency.c   | 2 +-
+ test/midiclock.c | 4 ++--
+ test/mm.c        | 8 ++++----
+ test/sysex.c     | 2 +-
+ test/test.c      | 2 +-
+ 5 files changed, 9 insertions(+), 9 deletions(-)
+
+diff --git a/test/latency.c b/test/latency.c
+index 507648c..a3d5b4c 100755
+--- a/test/latency.c
++++ b/test/latency.c
+@@ -280,7 +280,7 @@ int get_number(char *prompt)
+ {
+     char line[STRING_MAX];
+     int n = 0, i;
+-    printf(prompt);
++    fputs(prompt, stdout);
+     while (n != 1) {
+         n = scanf("%d", &i);
+         fgets(line, STRING_MAX, stdin);
+diff --git a/test/midiclock.c b/test/midiclock.c
+index 60fcf7a..4031ab0 100644
+--- a/test/midiclock.c
++++ b/test/midiclock.c
+@@ -167,7 +167,7 @@ int get_number(char *prompt)
+ {
+     char line[STRING_MAX];
+     int n = 0, i;
+-    printf(prompt);
++    fputs(prompt, stdout);
+     while (n != 1) {
+         n = scanf("%d", &i);
+         fgets(line, STRING_MAX, stdin);
+@@ -256,7 +256,7 @@ int main(int argc, char **argv)
+     err = Pm_OpenOutput(&midi, outp, DRIVER_INFO, OUTPUT_BUFFER_SIZE, 
+                         TIME_PROC, TIME_INFO, LATENCY);
+     if (err) {
+-        printf(Pm_GetErrorText(err));
++        puts(Pm_GetErrorText(err));
```

Comment:
`puts` will add additional `\n` at the end of the string. Applying only this patch will cause inconsistency. There might be other changes in `Pm_GetErrorText` which should be patched together.